src/Routes.js

```diff
@@ -20,7 +21,8 @@ InsightsRoute.propTypes = {
 
 export const routes = {
     home: '/',
-    details: '/:id'
+    details: '/:id',
+    finditfixit: '/fixit'
```

Comment:
I assume eventually there will be a separate page for each playbook and this will be parameterized (e.g. `/:id/fixit`)

Reply:
OK that makes sense.